compose.yml

```
@@ -0,0 +1,11 @@
+services:
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)